README.md

```diff
@@ -0,0 +1,614 @@
+# API Advisor
+
+```json
+"baseURL": "https://csgo-dashboard-rank.herokuapp.com"
+```
+---
+## Summary
+- [Players](#players)
+  - [Register players](#register-players)
+  - [Register a player with an params](#player-parms)
+  - [Register a player without an http](#player-http)
+  - [Errors](#errors-players)
+    - [Register a player with an incorrect url](#register-player-with-incorrect-url)
+    - [Register a already existing player](#player-added)
+    - [Register a player that was not found](#player-found)
+  - [Display all registered players](#display-players)
+- [Matches](#matches) 
+  - [Register matches](#register-matches)
+  - [Register a match with an params](#match-parms)
+  - [Register a match without an http](#match-http)
+  - [Errors](#errors-matches)
+    - [Register match with an incorrect url](#register-match-with-incorrect-url)
+    - [Register an already existing match](#register-already-match)
+    - [Register a match that was not found](#match-found)
+  - [Display all registered matches](#display-match)
+- [Ranks](#ranks)
+  - [Kills](#rank-kills-sorted)
+    - [Query params](#query-params)
```

Comment:
The id should be `#rank-kills-query-params` since we can (and will) have query params to other endpoints